### PR TITLE
Update/cleanup routes

### PR DESCRIPTION
**Pull request by ccali11**
@shanejearley - Cleaned up routes per discussion in #271.

### CODE REVIEW

**Comment by ccali11** [comment]
Also added `browserify` package.

**Comment by ccali11** [comment]
^Made one more quick fix to `LoginCredentials`.

File: services/users/src/routes/signup.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import express from 'express'
```
Comment (shanejearley) [inline services/users/src/routes/signup.ts:1]: Can this file go?

**Review by shanejearley — APPROVED** [review]
Great cleanup. Not sure about that signup.ts file.

File: services/users/src/routes/signup.ts (hunk 1 above)
Comment (shanejearley) [reply services/users/src/routes/signup.ts:1]: If so, maybe remove in the next PR.

**Comment by ccali11** [comment]
Did I leave an extra file in there? 😬

**Pull request merged**